napari/layers/shapes/_shapes_models/_polygon_base.py

```diff
@@ -80,6 +80,12 @@ def data(self, data):
             )
 
         self._data = data
+        self._bounding_box = np.round(
```

Comment:
This should not be rounded. (e.g. some datasets may be entirely in [0, 1]).

Reply:
Moved round again in proper place